recipes/reentry/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "reentry" %}
+{% set version = "1.0.2"%}
+{% set sha256 = "d981bf90c46a11819ff35bea227e4337019ae3b7719038b51e0e21180321c656"%}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  #fn: {{ name }}-{{ version }}.tar.gz
+  #url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  #sha256: {{ sha256 }}
+  git_url: https://github.com/DropD/reentry.git
```

Comment:
Instead of pulling the repo from github, can you use the release archive posted there instead? See:

http://conda-forge.github.io/docs/meta.html#build-from-tarballs-not-repos

The tarball can be found at:

https://github.com/DropD/reentry/releases